### ISSUE TITLE
add pause() to sound playback

### ISSUE THREAD

**Issue by REAS** [processing/processing-sound-archive#6]

**Comment by shiffman** [comment]
@wirsing yes, I think I meant this for sound files, it would be nice to be able to pause them (and restart) without having to `stop()`?  I don't really recall the exact scenario this came up.

**Comment by wirsing** [comment]
@shiffman that makes sense, i'll implement it soon!
